aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst

```diff
@@ -0,0 +1,248 @@
+Deploy an Ubuntu Pro EKS cluster
+================================
+
+This tutorial shows how to deploy an EKS Cluster with an Ubuntu Pro node.
+
+What approaches are there
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Go to the Cloud-init section if you use Ubuntu Pro service without the need to reboot the image, else go to the Packer section.
```

Comment:
the section is called "What approaches are there", but there no answer is provided. This sentence doesn't add value here and doesn't help readers who don't know why or when a reboot is needed